alpine-38.dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM alpine:3.8
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# apt-utils seems missing and warnings are shown, so we install it.
+RUN apk update && \ 
+ echo 'UTC' > /etc/timezone && \
+ apk add tzdata file bash && \
+ rm /etc/localtime
```

Comment:
This is necessary only on Ubuntu, to force `dpkg-reconfigure tzdata` to do stuff.

Reply:
removed the line